<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CattleBuyScreen</class>
 <widget class="QDialog" name="CattleBuyScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tela de Compra de Gado</string>
  </property>
  <widget class="QLabel" name="labelEarring">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Brinco:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBreed">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Raça</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDateA">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Data de aquisição</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDateB">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>240</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Data de nascimento</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelWeight">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>310</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Peso</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPrice">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>310</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Valor</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Giro de Gado</string>
   </property>
  </widget>
  <widget class="QPushButton" name="registerButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>470</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cadastrar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>470</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sair</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputBreed">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>591</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputDateA">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>260</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputDateB">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>260</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputWeight">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>330</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputPrice">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>330</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="labelCattleEarring">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>#10</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
